backends/backends.c

```diff
@@ -668,7 +687,20 @@ void *backends_main(void *ptr) {
                 while(sock != -1 && errno != EWOULDBLOCK) {
                     buffer_need_bytes(response, 4096);
 
+                    ssize_t r;
+#ifdef ENABLE_HTTPS
+                    if(opentsdb_ssl.conn) {
+                        if(!opentsdb_ssl.flags) {
+                            r = SSL_read(opentsdb_ssl.conn,&response->buffer[response->len], response->size - response->len);
+                        } else {
+                            r = recv(sock, &response->buffer[response->len], response->size - response->len, MSG_DONTWAIT);
+                        }
+                    } else {
+                        r = recv(sock, &response->buffer[response->len], response->size - response->len, MSG_DONTWAIT);
+                    }
+#else
                     ssize_t r = recv(sock, &response->buffer[response->len], response->size - response->len, MSG_DONTWAIT);
```

Comment:
3 * `r = recv(sock, &response->buffer[response->len], response->size - response->len, MSG_DONTWAIT);`

Reply:
error: redeclaration of ‘r’ with no linkage

Reply:
I created an PR previously for this error, and i do not have more it here!

Reply:
I fixed it now!